docs/tutorials/my-devices-registration.md

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
hi i took a look at your tutorial and i think its lacking a bunch of stuff…

there is not alot of context here.  if someone comes to the docs.waggle website this tutorial is not really related to the other top level tutorials “access waggle sesnors”, “edge apps”, etc.

I think we want to create a top level tutorial item: “Building your own Waggle Device”.  And this page should outline the following:
- Why someone would want to create their own waggle device ?  (for local development, classroom, etc.)
- List of supported platforms (right now only the Nvidia Nano)
- Links to the node-platforms repo for the Nvidia Nano
- Then a link to “my devices” where u (wil eventually) be able to see all your devices that you have previously registered

this needs to a full tutorial… assuming the user knows nothing and has no context about really what “making their own waggle devices is”